docs/docs/interpreter/ignite.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "Ignite Interpreter"
+description: "Ignite user guide"
+group: manual
+---
+{% include JB/setup %}
+
+## Ignite Interpreter for Apache Zeppelin
+
+### Overview
+[Apache Ignite](https://ignite.apache.org/) In-Memory Data Fabric is a high-performance, integrated and distributed in-memory platform for computing and transacting on large-scale data sets in real-time, orders of magnitude faster than possible with traditional disk-based or flash technologies.
+
+![Apache Ignite](/assets/themes/zeppelin/img/docs-img/ignite-logo.png)
+
+You can use Zeppelin to retrieve distributed data from cache using Ignite SQL interpreter. Moreover, Ignite interpreter allows you to execute any Scala code in cases when SQL doesn't fit to your requirements. For example, you can populate data into your caches or execute distributed computations.
+
+### Installing and Running Ignite example
+In order to use Ignite interpreters, you may install Apache Ignite in some simple steps:
+
+  1. Download Ignite [source release](https://ignite.apache.org/download.html#sources) or [binary release](https://ignite.apache.org/download.html#binaries) whatever you want. But you must download Ignite as the same version of Zeppelin's. If it is not, you can't use scala code on Zeppelin. You can find ignite version in Zepplin at the pom.xml which is placed under `path/to/your-Zeppelin/ignite/pom.xml` . please check `ignite.version` .
```

Comment:
I see. 
So I check that is there a way a user who has Zeppelin binary release can check the ignite version. However, currently in Zeppelin binary release( zeppelin-0.5.0-bin-spark-xxx-hadoop-xx ), ignite is not included. Only source release include ignite. 
People said that Zeppelin next binary release will include ignite, too. So I added an explanation about this.